src/include/sof/audio/module_adapter/module/modules.h

```diff
@@ -75,6 +73,7 @@ static inline void declare_dynamic_module_adapter(struct comp_driver *drv,
 	drv->ops.dai_ts_start = module_adapter_ts_start_op;
 	drv->ops.dai_ts_stop = module_adapter_ts_stop_op;
 	drv->ops.dai_ts_get = module_adapter_ts_get_op;
+	drv->adapter_ops = &processing_module_adapter_interface;
 }
```

Comment:
this function is rather long and is only called in one location. Can we move it either to a normal static function in that file, or to an exported function in modules.c, then we wouldn't need to make `processing_module_adapter_interface` global

Reply:
@lyakh I can move the `declare_dynamic_module_adapter` function to lib_manager where it is used, but we still need global `processing_module_adapter_interface`.

Reply:
...or you can move it to the same file where `processing_module_adapter_interface` is defined, then you keep it `static` and make the function exported instead. I think global functions are usually preferred over global data.